runtime/karura/Cargo.toml

```diff
@@ -323,3 +326,5 @@ try-runtime = [
 	"module-homa-xcm/try-runtime",
 	"module-session-manager/try-runtime",
 ]
+
+integration-tests = []
```

Comment:
why is this needed?

Reply:
integration-tests use some special config only in testcase, but it's not required for production config. so I use `integration-tests` feature flag in runtime